contracts/prize-pool/barnbridge/BarnPrizePool.sol

```diff
@@ -132,4 +149,23 @@ contract BarnPrizePool is PrizePool {
 
         return owed.add(pendingReward);
     }
+
+    function withdrawSplitReserve() external onlyReserve returns (uint256) {
+        uint256 amount = reserveTotalSupply;
+        reserveTotalSupply = 0;
+        uint256 redeemed = _redeem(amount);
```

Comment:
If redeemed is an odd number 1 wei of the token will be left in the contract (unclaimable). Maybe just do something like:
```
uint256 PTReserve = redeemed.div(2);
uint256 BarnReserve = redeemed.sub(PTReserve);
.safeTransfer(addressofPT, PTReserve);
.safeTransfer(addressofBarn, BarnReserve);
```

Reply:
Done!